exchanges/kraken/kraken_futures.go

```diff
@@ -292,6 +293,10 @@ func (k *Kraken) SendFuturesAuthRequest(method, path string, postData url.Values
 		Verbose:       k.Verbose,
 		HTTPDebugging: k.HTTPDebugging,
 		HTTPRecording: k.HTTPRecording,
+	}
+
+	err := k.SendPayload(ctx, request.Unset, func() (*request.Item, error) {
+		return item, nil
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Kraken has unimplemented rate limits